ledger/participant-integration-api/src/main/scala/platform/configuration/InitialLedgerConfiguration.scala

```diff
@@ -22,17 +22,16 @@ import com.daml.ledger.configuration.{Configuration, LedgerTimeModel}
   * @param delayBeforeSubmitting The delay until the participant tries to submit a configuration.
   */
 final case class InitialLedgerConfiguration(
-    maxDeduplicationDuration: Duration,
-    avgTransactionLatency: Duration,
-    minSkew: Duration,
-    maxSkew: Duration,
-    delayBeforeSubmitting: Duration,
+    enabled: Boolean = true,
```

Comment:
I don't like the pattern of adding an `enabled` into a populated structure.  I would stick with the Option, or if this makes it harder to work with from a HOCON point of view then possibly make `InitialLedgerConfiguration` into a trait which has `On` and `Off` variants.